scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/entity/RecurringScheduleEntity.java

```diff
@@ -4,100 +4,110 @@
 import java.util.Arrays;
 import java.util.Date;
 
-import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.NamedQueries;
 import javax.persistence.NamedQuery;
 import javax.persistence.Table;
 import javax.validation.constraints.NotNull;
 
+import org.cloudfoundry.autoscaler.scheduler.util.DateDeserializer;
+import org.cloudfoundry.autoscaler.scheduler.util.DateHelper;
+import org.cloudfoundry.autoscaler.scheduler.util.DateSerializer;
+import org.cloudfoundry.autoscaler.scheduler.util.SqlTimeDeserializer;
+import org.cloudfoundry.autoscaler.scheduler.util.SqlTimeSerializer;
 import org.hibernate.annotations.Type;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 
 @Entity
 @Table(name = "app_scaling_recurring_schedule")
 @NamedQueries({
 		@NamedQuery(name = RecurringScheduleEntity.query_recurringSchedulesByAppId, query = RecurringScheduleEntity.jpql_recurringSchedulesByAppId) })
 public class RecurringScheduleEntity extends ScheduleEntity {
 
-	@Column(name = "start_time")
+	@JsonDeserialize(using = SqlTimeDeserializer.class)
+	@JsonSerialize(using = SqlTimeSerializer.class)
 	@NotNull
-	private Time startTime;
+	private Time start_time;

```

Comment:
The internal names should have remained as startTime. This isn't conventional Java code.